tutorials/beginner/introduction.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>Seems to be an extra base in the first two words</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I'm not sure what you mean.